internal/delete/actions/csi_volumesnapshot_action.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2020 the Velero contributors.
```

Comment:
Should this way better --- we add a `csi` subfolder, and put all the CSI plugin files there and remove the `csi_` prefix from the files?

Reply:
How about deleting the csi prefix?

Reply:
Modified.